src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -0,0 +1,497 @@
+    location,
+    onNihStatusUpdate,
+    partialSave,
+    researcher,
+    showValidationMessages,
+    nextPage,
+    cloudProviderType,
+    cloudProvider,
+    isCloudUseInvalid,
+    ariaLevel = 2
+  } = props;
+
+  const navButtonContainerStyle = {
+    marginTop: '5rem'
+  };
+
+  const formatSOString = (name, email) => {
+    if(isEmpty(name)) { return '';}
+    const nameString = `${name}`;
+    const emailString = !isNil(email) ? ` (${email})` : '';
+    return nameString + emailString;
+  };
+
+  //initial state variable assignment
+  const [checkCollaborator, setCheckCollaborator] = useState(props.checkCollaborator);//
+  const [checkNihDataOnly, setCheckNihDataOnly] = useState(props.checkNihDataOnly);//
+  const [signingOfficial, setSigningOfficial] = useState();//
+  const [itDirector, setITDirector] = useState(props.itDirector || '');//
+  const [principalInvestigator, setPrincipalInvestigator] = useState(props.principalInvestigator || '');//
+  const [anvilUse, setAnvilUse] = useState(props.anvilUse || '');
+  const [cloudUse, setCloudUse] = useState(props.cloudUse || '');
+  const [localUse, setLocalUse] = useState(props.localUse || '');
+  const [researcherUser, setResearcherUser] = useState(props.researcherUser);//
+  const [libraryCardReqSatisfied, setLibraryCardReqSatisfied] = useState(false);
+
+  useEffect(() => {
+    setLibraryCardReqSatisfied(!isEmpty(get('libraryCards')(researcherUser)) || checkNihDataOnly);
+  }, [checkNihDataOnly, researcherUser]);
+
+  useEffect(() => {
+    setSigningOfficial(props.signingOfficial);
+    setCheckCollaborator(props.checkCollaborator);
+    setCheckNihDataOnly(props.checkNihDataOnly);
+    setITDirector(props.itDirector);
+    setPrincipalInvestigator(props.principalInvestigator);
+    setAnvilUse(props.anvilUse);
+    setCloudUse(props.cloudUse);
+    setLocalUse(props.localUse);
+    setResearcherUser(props.researcherUser);
+  }, [props.signingOfficial, props.checkCollaborator, props.itDirector, props.principalInvestigator, props.anvilUse, props.cloudUse, props.localUse, props.researcherUser, props.checkNihDataOnly]);
+
+  return (
+    div({
+      datacy: 'researcher-info',
+      className: 'col-lg-10 col-lg-offset-1 col-md-12 col-sm-12 col-xs-12' }, [
+
+      div({ style: { backgroundColor: 'rgba(184, 205, 211, .15)', padding: '20px 30px' } }, [
+        div({
+          datacy: 'researcher-info-profile-submitted',
+          isRendered: (completed === true && libraryCardReqSatisfied), className: 'rp-alert' }, [
+          Alert({
+            id: 'profileSubmitted',
+            type: 'important',
+            title: span([
+              `You must submit `, profileLink, ` and obtain a `, libraryCardLink,
+              ` from your Signing official before you can submit a Data Access Request.`
+            ])
+          })
+        ]),
+
+        h2('Step 1: Researcher Information'),
+
+        div({className: 'dar-application-row'}, [
+          h(FormField, {
```

Comment:
You can disable the field here since we're pre-filling the field with the user profile name